src/CoreShop/Bundle/RuleBundle/EventListener/AvailabilityCheckMaintenanceListener.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * CoreShop.
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2015-2017 Dominik Pfaffenbauer (https://www.pfaffenbauer.at)
+ * @license    https://www.coreshop.org/license     GNU General Public License version 3 (GPLv3)
+ */
+
+namespace CoreShop\Bundle\CoreBundle\EventListener\Rule;
+
+use CoreShop\Bundle\RuleBundle\Accessor\RuleAvailabilityProcessorInterface;
+use CoreShop\Component\Core\Configuration\ConfigurationServiceInterface;
+use Pimcore\Event\System\MaintenanceEvent;
+use Pimcore\Model\Schedule\Maintenance\Job;
+
+final class AvailabilityCheckMaintenanceListener
```

Comment:
service is declared in RuleBundle, move declaration to CoreBundle.

BTW: This service needs to stay in CoreBundle cause it uses the Configuration Service which is not a valid dependency in RuleBundle